uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -374,15 +376,54 @@ sub determineSubjectID {
                             $scannerID,
                             $this->{dbhr}
                         );
+                        
+    #==============================================#
+    # Retrieve the candidate's project ID (if any) #
+    # and session's subproject ID                  #
+    #==============================================#
+    (my $query = <<QUERY) =~ s/\n/ /gm;
+    SELECT
+      c.ProjectID, s.SubprojectID
+    FROM
+      session s 
+    JOIN candidate c USING(CandID)
+    WHERE
+      s.Visit_label = ?
+    AND
+      s.CandID = ?
```

Comment:
Should not you use where c.Active = 'Y' and s.Active = 'Y'?   c.active can probably be omitted as the candidate should be active to add MRI.

Reply:
Good point. And yes, we should also add `c.active='Y'` since a candidate can be scanned and be inactivated afterwards.

Reply:
Yes it make sense. I wasn't really sure myself if active flag where important here.